applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -778,7 +779,7 @@ static void stream_sent_cb(struct bt_bap_stream *stream)
 
 	ret = channel_index_get(stream->conn, &channel_index);
 	if (ret) {
-		LOG_ERR("Channel index not found");
+		LOG_WRN("Channel index not found");
```

Comment:
Why this change?

Reply:
For debugging purposes. Should not be included.